app/main.py

```diff
@@ -255,15 +255,19 @@ def update_live_alerts_data(
     already_loaded_frames,
     site_devices_data,
     night_time_data,
+    user_headers,
 ):
     """
     This is the key callback of the platform. Triggered by the interval component, it queries the database via the API
     client so as to identify live alerts, load the associated data and trigger their display on the platform. This doc-
     string should be completed but more details can be found in the comments below.
     """
 
+    if user_headers is None:
+        raise PreventUpdate
+
     # Fetching live alerts where is_acknowledged is False
-    response = api_client.get_ongoing_alerts()
+    response = requests.get(cfg.API_URL + "/alerts/ongoing", headers=user_headers)
```

Comment:
let's keep the API client, much safer (it will be updated if routes get updated)
We can update the token used by the client as such
```python
api_client.token = user_token
```

Reply:
Good idea, thanks!